joomla/joomla-cms
# [fix]Migration from 2.5.28 to 3.6 returns Fatal errors

Pull Request for Issue #11117 .
#### Summary of Changes

set $this->input if you update from 2.5 to 3.x
#### Why

Since 3.5 we need to relogin if we update from an old version. But only if you relogin the old files get deleted.
The relogin fails because of files that did not removed yet.
#### Testing Instructions

install 2.5
Update to 3.6
see the issue

```
Notice: Undefined property: LoginController::$input in ...administrator\components\com_login\controller.php on line 36
Fatal error: Call to a member function set() on null in ...administrator\components\com_login\controller.php on line 36
```

roll back to 2.5
install the update to 3.6 with the changes in this patch (i'm going to prepare a update server and package later)
there should be no issues
